openodia/_understandData.py

```diff
@@ -7,20 +8,40 @@
 
 class UnderstandData:
     """Tokenizer implementation"""
-
+    
     @classmethod
-    def word_tokenizer(cls, text):
+    def word_tokenizer(cls,text):
         """Split the text into words"""
-        # TODO: Do not remove the punctuation characters
+        punct = regex.compile(r'(\p{Punctuation})')    
+        x=punct.split(test_data)
+        str1=""
+        for i in x:
+            str1+=i+' '
         token_list = [
-            token
-            for token in re.split(
-                r"[!\"#$%&\'()*+,-./:;<=>?@[\\\]\^\_`{|}~୲—\s+]", text, flags=re.UNICODE
-            )
-            if token
-        ]
+                token
+                for token in str1.split()
+                if token
+            ]
         return token_list
 
+# import regex
+
+# def word_tokenizer(text):
+#     punct = regex.compile(r'(\p{Punctuation})')    
+#     x=punct.split(test_data)
+#     str1=""
+#     for i in x:
+#         str1+=i+' '
+#     token_list = [
+#             token
+#             for token in str1.split()
+#             if token
+#         ]
+#     return token_list
+
+# test_data = "ଭାରତୀୟ ସର୍ବୋଚ୍ଚ ନ୍ୟାୟାଳୟ, ଭାରତର ଉଚ୍ଚତମ ନ୍ୟାୟିକ ଅନୁଷ୍ଠାନ ଅଟେ ।"
+# print(word_tokenizer(test_data))
```

Comment:
Commented code should not be present in the code.
If past code is needed, can be fetched from `git` commit history.
Keeping commented code reduces readability factor.